distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Filter_Condition_Helpers.enso

```diff
@@ -78,7 +79,7 @@ make_filter_column source_column:Column filter_condition on_problems:Problem_Beh
         Is_In values _ ->
             warn_on_nothing_in_comparison_vector filter_condition values <|
                 source_column.is_in values
-        Predicate p -> Column.from_vector source_column.name (source_column.to_vector.filter p)
+        Predicate p -> Error.throw (Illegal_State.Error "Predicate filter conditions need to be resolved before being used as a filter column.")
```

Comment:
- _"need to be resolved"_ is not clear to me...
- how does one "resolve" `Filter_Condition.Predicate`?

Reply:
All the callers to make_filter_column are resolving predicates to a mask column. 

The internal method can't do this.